### PR TITLE
[github-actions] add pretty check

### PR DESCRIPTION
**Pull request by gjc13**
Github actions provide more coherent machines and will boost the CI check.

### CODE REVIEW

**Comment by gjc13** [comment]
@bukepo @jwhui 
Since we cannot check the action status if github actions is not enabled on master. We may merge this PR first and add all the check afterwards.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      if: "github.ref != 'refs/heads/master'"
+
+  pretty:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Bootstrap
+      run: sudo OS_NAME=linux BUILD_TARGET=pretty-check tests/scripts/bootstrap.sh
```
Comment (bukepo) [inline .github/workflows/build.yml]:
```suggestion
      run: OS_NAME=linux BUILD_TARGET=pretty-check tests/scripts/bootstrap.sh
```

File: tests/scripts/bootstrap.sh [hunk 2]
```diff
@@ -73,7 +73,7 @@ configure_network() {
     echo 1 | sudo tee /proc/sys/net/ipv4/conf/all/forwarding
 }
 
-case $TRAVIS_OS_NAME in
+case $OS_NAME in
```
Comment (bukepo) [inline tests/scripts/bootstrap.sh:76]: use a well-known environment?
https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script

File: .travis.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ cache:
   directories:
   - $HOME/.cache/tools
 
-before_install: .travis/before_install.sh
+before_install: OS_NAME=${TRAVIS_OS_NAME} ./tests/scripts/bootstrap.sh
```
Comment (jwhui) [inline .travis.yml:9]: Do we still need `OS_NAME` here?

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+  pretty:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Bootstrap
+      run: sudo BUILD_TARGET=pretty-check tests/scripts/bootstrap.sh
+    - name: Check
+      run: script/make-pretty check
+
```
Comment (jwhui) [inline .github/workflows/build.yml:51]: nit: extra line

**Pull request merged**